docs/extensions.rst

```diff
@@ -133,3 +133,38 @@ TLS
 ---
 
 See :doc:`specs/tls`.
+
+HTTP Trailers
+---
+
+The Trailer response header allows the sender to include additional fields at the
+end of chunked messages in order to supply metadata that might be dynamically
+generated while the message body is sent, such as a message integrity check,
+digital signature, or post-processing status.
+
+ASGI servers that implement this extension will provide
+``http.response.trailers`` in the extensions part of the scope::
+
+    "scope": {
+        ...
+        "extensions": {
+            "http.response.trailers": {},
+        },
+    }
+
+An ASGI framework interested in sending trailing headers to the client, must include
+the ``Trailer`` header in the ``http.response.start``. That will allow the ASGI server
```

Comment:
Is the `Trailer` header the same as `TE: trailers` mentioned below? Or should there be an additional field?

Reply:
No, this `Trailer` header is a response header. The `TE: trailers` below is from the request header.

Example of full interaction:
- The client sends a `TE: trailers` request header.
- The server sends `Trailer: X-Server-Timing` in the response headers.
- Body is sent.
- `X-Server-Timing` is sent as trailing headers.